applications/DEMApplication/python_scripts/DEM_procedures.py

```diff
@@ -478,6 +478,9 @@ def AddSpheresVariables(self, model_part, DEM_parameters):
         model_part.AddNodalSolutionStepVariable(NEIGHBOUR_SIZE)
         model_part.AddNodalSolutionStepVariable(NEIGHBOUR_RATIO)
 
+        # For averaging forces when substepping
+        model_part.AddNodalSolutionStepVariable(CONTACT_IMPULSE)
```

Comment:
This variable should not be added in general for all DEM computations. Can you add it in a coupling file instead of here?

Reply:
Solved now